log/logger.go

```diff
@@ -0,0 +1,56 @@
+package log
+
+import (
+	"os"
+	"sync"
+
+	"gopkg.in/natefinch/lumberjack.v2"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// TODO: implement a mechanism to load this values from configuration
+const (
+	logFilePath = "temp/logs/yhs.log"
```

Comment:
Yeah sure.

Reply:
Hi, @richscott. I just realized that in the current implementation, I used `lumberjack` for log file rotation. If one log file becomes too large, it will create a new file and rename the previous one. 

Therefore, we need to keep the logs at least one directory deep. Otherwise, it will start writing multiple log files in the root directory. Do you have any suggestions for a better directory name? I'm not too fond of `temp/something` 😅